cmd/etcd.go

```diff
@@ -75,56 +82,43 @@ func newDelKeyCommand() *cobra.Command {
 // newPutKeyCommand returns a put key subcommand of EtcdCommand.
 func newPutKeyCommand() *cobra.Command {
 	m := &cobra.Command{
-		Use:   "putkey",
-		Short: "[ONLY FOR TEST!] put a key in the path of TiDB schema versions by `putkey [key] [value]`",
+		Use:   "putkey <key> <value>",
+		Short: "[ONLY FOR TEST] put a key in the path of TiDB schema versions by `putkey [key] [value]`",
 		Run:   putKeyCommandFunc,
 	}
 	return m
 }
 
+// newEvictOwnerCommand returns a evict owner subcommand of EtcdCommand.
+func newEvictOwnerCommand() *cobra.Command {
+	m := &cobra.Command{
+		Use:   "evictowner",
```

Comment:
Could we update this comment?
Maybe this key isn't the owner, it's only a DDL worker.

Reply:
Why this key is not the owner?
IMO, this  key must be owner.

Reply:
This key's path prefix has "owner"，it uses to campaign the owner. But it doesn't mean that the key with this prefix is the owner.
3 TiDB nodes:
```
# ETCDCTL_API=3 etcdctl --endpoints=127.0.0.1:42379 get /tidb/ddl  --prefix
/tidb/ddl/all_schema_versions/1ade5078-57d9-439f-bb29-0fc9daa3c095
15
/tidb/ddl/all_schema_versions/27509709-fe10-492b-9c37-19aea6492233
15
/tidb/ddl/all_schema_versions/aa6b2dc0-922c-4b5d-885b-eb64907002a5
15
/tidb/ddl/fg/owner/12c8697feef62d5f
27509709-fe10-492b-9c37-19aea6492233
/tidb/ddl/fg/owner/12c8697feef62d73
aa6b2dc0-922c-4b5d-885b-eb64907002a5
/tidb/ddl/fg/owner/12c8697feef62d85
1ade5078-57d9-439f-bb29-0fc9daa3c095
/tidb/ddl/global_schema_version
0
```